helix-term/src/ui/picker.rs

```diff
@@ -263,21 +311,63 @@ impl<T: 'static> Component for Picker<T> {
         self.prompt.render(area, surface, cx);
 
         // -- Separator
```

Comment:
Can you draw an ascii diagram how it will look like here? As stuff gets more complicated an ascii diagram will be more useful.

Reply:
That's a good idea, I'll add it.